gr-digital/lib/costas_loop_cc_impl.cc

```diff
@@ -175,6 +179,15 @@ int costas_loop_cc_impl::work(int noutput_items,
                       nitems_read(0) + noutput_items,
                       pmt::intern("phase_est"));
 
+    // Get this out of the for loop if not used:
+    bool hasAdditionalOutputs = false;
+    if (freq_optr != NULL)
```

Comment:
why directly compare with `NULL`? Why not `if (freq_optr)`? The compiler output code should be identical, and it is, at least in my opinion and experience, easier to read. either way works though for this purpose.

Reply:
@michaelld, the throw() was original from master.  If I remove it, is there a new standard being used in master to raise the error state?  I can incorporate it here.  I did update all the !=NULL references for readability so when I push a new update here it'll be more readable.